Documentation/VSTestIntegration.md

```diff
@@ -87,6 +87,6 @@ The datacollectors will be bundled as a separate NuGet package, the reference to
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
     <PackageReference Include="MSTest.TestAdapter" Version="1.3.2" />
     <PackageReference Include="MSTest.TestFramework" Version="1.3.2" />
-    <PackageReference Include="coverlet.collector" Version="1.0.0" />   
+    <PackageReference Include="coverlet.collector" Version="1.0.0" /> 
```

Comment:
@vagisha-nidhi in my test it works with
```
  <ItemGroup>
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.1.0" />
    <PackageReference Include="coverlet.collector" Version="1.0.0" />
    <PackageReference Include="xunit" Version="2.4.1" />
    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1">
      <PrivateAssets>all</PrivateAssets>
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
    </PackageReference>
  </ItemGroup>
```
Does it make sense add "number version"? Maybe something like
```
<ItemGroup>
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="x.x.x" />
    <PackageReference Include="MSTest.TestAdapter" Version="x.x.x" />
    <PackageReference Include="MSTest.TestFramework" Version="x.x.x" />
    <PackageReference Include="coverlet.collector" Version="1.0.0" /> 
</ItemGroup>
```

Reply:
@MarcoRossignoli  Yeah. The versions are updated as when required. So its better to have a variable so not to confuse(as if it will work only with the particular version mentioned in the documentation).